fvm/transactionVerifier.go

```diff
@@ -191,16 +191,33 @@ func (v *TransactionSignatureVerifier) verifyAccountSignature(
 		}
 		return nil, errors.NewInvalidPayloadSignatureError(txSig.Address, txSig.KeyIndex, err)
 	}
+	if validWOTag {
+		return &accountKey, nil
+	}
 
-	if !valid {
-		err = fmt.Errorf("signature is not valid")
+	validWTag, err := v.SignatureVerifier.Verify(
```

Comment:
```suggestion
	validWithTag, err := v.SignatureVerifier.Verify(
```

Reply:
addressed in 7a7ff10d04ac2a1179285968dc2db8d7d184fefc